packages/calling/src/CallingClient/line/line.test.ts

```diff
@@ -154,4 +157,104 @@ describe('Line Tests', () => {
       expect(line.getRegistrationStatus()).toEqual(MobiusStatus.DEFAULT);
     });
   });
+  describe('Line calling tests', () => {
+    let line;
+
+    beforeEach(() => {
+      line = new Line(
+        userId,
+        clientDeviceUri,
+        LineStatus.ACTIVE,
+        mutex,
+        primaryMobiusUris(),
+        backupMobiusUris(),
+        LOGGER.INFO
+      );
+    });
+
+    afterEach(() => {
+      jest.clearAllTimers();
+      jest.clearAllMocks();
+      jest.useRealTimers();
+      line.removeAllListeners();
+    });
+    it('Return a successful call object while making call', () => {
+      const call = line.makeCall({address: '5003', type: CallType.URI});
+
+      expect(call).toBeTruthy();
+      expect(line.getCall(call ? call.getCorrelationId() : '')).toBe(call);
+      expect(call ? call.direction : undefined).toStrictEqual(CallDirection.OUTBOUND);
+      call?.end();
+    });
+
+    it('Return a successful call object while making call to FAC codes', () => {
+      const call = line.makeCall({address: '*25', type: CallType.URI});
+
+      expect(call).toBeTruthy();
+      expect(call ? call.direction : undefined).toStrictEqual(CallDirection.OUTBOUND);
+      call?.end();
+    });
+
+    it('Remove spaces from dialled number while making call', () => {
+      const call = line.makeCall({address: '+91 123 456 7890', type: CallType.URI});
+
+      expect(call).toBeTruthy();
+      expect(call ? call.direction : undefined).toStrictEqual(CallDirection.OUTBOUND);
+      expect(call ? call.destination.address : undefined).toStrictEqual('tel:+911234567890');
+      call?.end();
+    });
+
+    it('Remove hyphen from dialled number while making call', () => {
+      const call = line.makeCall({address: '123-456-7890', type: CallType.URI});
+
+      expect(call).toBeTruthy();
+      expect(call ? call['direction'] : undefined).toStrictEqual(CallDirection.OUTBOUND);
+      expect(call ? call['destination']['address'] : undefined).toStrictEqual('tel:1234567890');
+      call?.end();
+    });
+
+    it('attempt to create call with incorrect number format 1', (done) => {
+      // There may be other listeners , which may create race
+      line.removeAllListeners(LINE_EVENTS.ERROR);
+      const createCallSpy = jest.spyOn(line.callManager, 'createCall');
+
+      line.on(LINE_EVENTS.ERROR, (error) => {
+        expect(error.message).toBe(
+          'An invalid phone number was detected. Check the number and try again.'
+        );
+        done();
+      });
+      try {
+        const call = line.makeCall({address: 'select#$@^^', type: CallType.URI});
+
+        expect(call).toBeUndefined();
+        expect(createCallSpy).toBeCalledTimes(0);
```

Comment:
replace toBeCalledTimes(0) with not.toBeCalledOncewith

Reply:
Changed.